### DISCUSSION TITLE
[Chapter 09] 최적의 HTTP 클라이언트 솔루션은?

### DISCUSSION THREAD

**Discussion by corock** [cowaine/spring-boot-for-msa#139] [Chapter 09. Application Integration: REST API] [answered]
9장에서는 스프링 프레임워크에서 서버 to 서버 통신을 위한 HTTP 클라이언트를 소개하고 있습니다.
`RestTemplate` 클래스와 `WebClient` 인터페이스 2가지가 언급되었는데, 혹시 이 두 가지 외에 많이 사용하는 HTTP 클라이언트가 있을까요?
`RestTemplate` 클래스는 [_deprecated_](https://www.youtube.com/watch?v=S4W3cJOuLrU) 된다는 괴담이 있었고, `WebClient` 인터페이스는 `webflux` 의존성을 바꿔야만 해서 둘 다 아쉽습니다.

최고의 HTTP 클라이언트를 찾고 싶습니다.

**Comment by JoisFe** [comment]
길웅님 회사에서 뭐 하나 쓰신다고 안했나요?

**Reply by coalong** [reply, marked as answer]
[우아한 feign 적용기](https://techblog.woowahan.com/2630/)

**Reply by Irisation23** [reply]
https://robin00q.tistory.com/71
